processor/signozlogspipelineprocessor/stanza/operator/operators/json/parser_test.go

```diff
@@ -98,6 +98,36 @@ func TestParser(t *testing.T) {
 				Body: `{"superkey":"superval"}`,
 			},
 		},
+		{
+			"recieved_map",
+			func(_ *Config) {},
+			&entry.Entry{
+				Body: map[string]any{
+					"superkey": "superval",
+					"superkey2": map[string]any{
+						"nestedkey1": "nestedval",
+					},
+					"superkey3": []string{"val1", "val2"},
+				},
+			},
+			&entry.Entry{
+				Attributes: map[string]any{
+					"superkey": "superval",
+					"superkey2": map[string]any{
+						"nestedkey1": "nestedval",
+					},
+					"superkey3": []string{"val1", "val2"},
+				},
+				// Note: body is not strigified by parser but the exporter when saving in clickhouse
+				Body: map[string]any{
```

Comment:
add a case for map[string]string , so that we are sure.

Reply:
Tested and every map based payload is received as map[string]any.

Even payload sent as map[int]float64 is received as map[string]any

Reply:
then it's good enough